packages/jaeger-ui/src/reducers/trace.js

```diff
@@ -124,6 +124,34 @@ function searchErred(state, { meta, payload }) {
   return { ...state, search };
 }
 
+function uploadStarted(state) {
+  const search = {
+    results: state.search.results,
```

Comment:
When keeping `results`, seems like `query` should also be retained?

Reply:
Yes, good catch!